.github/workflows/release-next-rspack.yml

```diff
@@ -0,0 +1,123 @@
+name: Release
```

Comment:
Regarding testing: You'll need to merge *something* as it's not possible to trigger a workflow that isn't in canary. So just merge something in this PR, and iterate on it in a later pr 👍.